### ISSUE TITLE
Consider NINA-W106 instead of 102.

### ISSUE THREAD

**Issue by flundstrom2** [edgepro-dev/edgepro-sch#1]
The 106 antenna is actually somewhat better than the 102, in both performance and robustness. The 106 has the same form-factor and is pad-compatible with the 102.

**Comment by jturnsek** [comment]
> The 106 antenna is actually somewhat better than the 102, in both performance and robustness. The 106 has the same form-factor and is pad-compatible with the 102.

I agree, but 106 antenna does need a lot more PCB around it, in order to function properly, which EdgePro1 board does not have.
